test/unittests/libp2p/net.cpp

```diff
@@ -439,6 +432,46 @@ BOOST_AUTO_TEST_CASE(kademlia)
     BOOST_REQUIRE_EQUAL(node.nodeTable->count(), 8);
 }
 
+BOOST_AUTO_TEST_CASE(hostNoCapsNoTcpListener)
+{
+	Host host("Test", NetworkConfig(c_localhostIp, randomPortNumber(), false /* upnp */, true /* allow local discovery */));
+	host.start();
+
+	// Wait 6 seconds for network to come up
+	uint32_t const step = 10;
+	for (unsigned i = 0; i < 6000; i += step)
+	{
+		this_thread::sleep_for(chrono::milliseconds(step));
+		if (host.haveNetwork())
+			break;
+	}
+
+    BOOST_REQUIRE(host.haveNetwork());
+    auto const hostPort = host.listenPort();
+    BOOST_REQUIRE(hostPort);
+	BOOST_REQUIRE(host.caps().empty());
```

Comment:
some tabs here

Reply:
Whoops, forgot to run git clang-format before comitting